.github/workflows/checkdocs.yml

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+# This workflow contains tests that operate on documentation files only. Some
+# checks modify the source so they cannot be combined into a single job.
+
+name: Documentation Checks
```

Comment:
I propose a shorter version. Maybe just drop the superfluous `Checks` part? Maybe call it just `Docs`?

![image](https://github.com/user-attachments/assets/9d3e926e-57d8-4466-b79b-63c6058a4610)